qemu/tests/cfg/msi_change_flag.cfg

```diff
@@ -0,0 +1,33 @@
+- msi_change_flag:
+    only virtio_net
+    type = msi_change_flag
+    #In this test need set snapshot for our test will chang guest msi support
+    image_snapshot = yes
+    filesize = 4000
+    file_trans_timeout = 2400
+    file_md5_check_timeout = 600
+    dd_cmd = "dd if=/dev/zero of=%s oflag=direct bs=1M count=%d"
+    Linux:
+        tmp_dir = "/var/tmp/"
+    Windows:
+        tmp_dir = "C:\\"
+        x86_64:
+        devcon_folder = "WIN_UTILS:\devcon\amd64\"
+    variants:
+        - disable_msi:
+            only Linux
+            pci_nomsi = yes
+            #after set pci_nomsi, we need kill the vm.
+            kill_vm = yes
+        - by_registry:
+            no Linux
+            msi_cmd = "reg add "HKLM\System\CurrentControlSet\Enum\%s\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v MSISupported /d %d /t REG_DWORD /f"
+            driver_name = netkvm
+            device_name = "Red Hat VirtIO Ethernet Adapter"
+    variants:
+        - file_transfer:
```

Comment:
I just want to see from the variable name what test is about. If I don't add this, I feel it's confused.

Reply:
If you will develop other cases that are not `file_transfer`, such as `netperf`/`ping`, you can add a "TODO" tag at first, otherwise I think renaming the test case is fine.